.github/actions/run-eden-test/action.yml

```diff
@@ -34,6 +34,7 @@ runs:
       if: always()
       uses: ./eden/.github/actions/publish-logs
       with:
+        report_name: eden-report-${{ inputs.suite }}-tpm-${{ inputs.tpm_enabled }}-${{ inputs.file_system }}
         file_system: ${{ inputs.file_system }}
```

Comment:
file_system and tpm_enabled can be removed from "with"

Reply:
fixed